### PR TITLE
Move scripts to demo folder

### PR DESCRIPTION
**Pull request by trbehera**
The scripts are primarily used for setting up the demo, hence it would
be better to move these scripts inside the demo folder.

Signed-off-by: Behera, Tushar Ranjan <tushar.ranjan.behera@intel.com>

### CODE REVIEW

**Review by KiranSukhavasi — APPROVED** [review]

File: Jenkinsfile.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ node('ubuntu18.04-OnDemand'){
   stage('build'){
     sh 'mvn clean install'
     print "Archive the artifacts"
-    sh 'cd component-samples && cp -r scripts demo && tar -czvf demo.tar.gz demo && mv demo.tar.gz ..'
+    sh 'cd component-samples && tar -czvf demo.tar.gz demo && mv demo.tar.gz ..'
     archiveArtifacts artifacts: 'demo.tar.gz', fingerprint: true, allowEmptyArchive: false
```
Comment (DukeDavis12) [inline Jenkinsfile.yml:15]: Line 37: scripts/keys_gen.sh  to demo/scripts/key_gen.sh

**Review by DukeDavis12 — CHANGES_REQUESTED** [review]
Please update line 37: Jenkinsfile.yml

File: Jenkinsfile.yml (hunk 1 above)
Comment (DukeDavis12) [reply Jenkinsfile.yml:15]: ++ line 38

Comment (trbehera) [reply Jenkinsfile.yml:15]: The scripts folder has been moved from component-samples/scripts to component-samples/demo/scripts, hence the above line isn't required.

Line 37 assumes that the scripts folder is available at component-samples/demo/scripts, hence no change required there.

Comment (DukeDavis12) [reply Jenkinsfile.yml:15]: Looks good 👍 .

**Review by DukeDavis12 — APPROVED** [review]

**Pull request merged**